en/micro-integrator/docs/administer-and-observe/logs/masking_sensitive_info_in_logs.md

```diff
@@ -1,54 +1,39 @@
+1.  Open the `log4j2.properties` file from the `<MI-HOME>/conf/` folder. and do the necessary changes. 
+2.  Add an extra `m` to the log file layout configuration (`layout.pattern` parameter) in which you want the values to be masked as shown below.
 
-    ```xml
-    log4j.appender.CARBON_CONSOLE.maskingPatternFile=path-to-masking-patterns
     ```
+    appender.CARBON_CONSOLE.layout.pattern = [%d] %5p {%c{1}} - %mm%ex%n
+    ```
+3. Update the [masking patterns](#the-masking-pattern-file) from the `<MI-HOME>/conf/deployment.toml` file as explained below.
 
-The `path-to-masking-patterns` value must be the absolute path to the [masking patterns file](#the-masking-pattern-file). In this file, each pattern is defined as key-value pairs (patten-name=pattern). 
-
-The following is a sample configuration for the above property:
-
-```xml
-log4j.appender.CARBON_CONSOLE.maskingPatternFile=/home/conf/masking-patterns.properties
-```
-
-For the `DailyRollingFileAppender` value, the above property would be similar to the following:
-
-```xml
-log4j.appender.CARBON_LOGFILE.maskingPatternFile=path-to-masking-patterns
-```
-
-In this case, you can define separate masking pattern files for the console appender and file appender (or configure only one property).
-
-## The masking pattern file
+## The masking patterns
 
-The masking pattern file is a property file that can contain one or more
-masking patterns. The following is a sample configuration that shows
-how to mask the credit card numbers in the logs.
+You can specify one or more masking patterns in the `<MI-HOME>/conf/deployment.toml` file. The following is a sample configuration that showcases how to mask the credit card numbers from the logs.
 
-```xml
-masking.pattern.sample.CREDIT_CARD_VISA=4[0-9]{6,}$
-masking.pattern.sample.CREDIT_CARD_MASTER=(?:5[1-5][0-9]{2}|222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}
-masking.pattern.sample.CREDIT_CARD_AMEX=[34|37][0-9]{14}$
+```toml
+[masking_pattern.properties]
+"CREDIT_CARD_VISA" = "4[0-9]{6,}$"
+"CREDIT_CARD_MASTER" = "(?:5[1-5][0-9]{2}|222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}"
+"CREDIT_CARD_AMEX" = "[34|37][0-9]{14}$"
 ```
 
-With this configuration, each log line is checked for all the configured
-patterns. If any match is found, it is masked with ‘*****’.
+With this configuration, each log line is checked for all the configured patterns. If any match is found, it is masked with ‘*****’.
 
 !!! Note
-    -   If the pattern file that is configured in the `log4j.properties` file
-    is not found, a default property file will be used (`           wso2-log-masking.properties          ` ).
+    -   If the pattern file that is configured in the `log4j2.properties` file
+    is not found, a default property file will be used (`wso2-log-masking.properties` ).
     -   If there are no patterns defined in the file, no masking happens.
-
-!!! Warning
```

Comment:
@Vathsan - Is this section valid anymore? Are we referring a log masking file from the log4j2.properties file?

Reply:
We can remove this note as this is not valid anymore.